### PR TITLE
hotfix/#143: Fixed tests for orders

### PR DESCRIPTION
**Pull request by Misha86**
Fixed tests for orders

### CODE REVIEW

**Review by pazuzu-ua — APPROVED** [review]

File: beauty/api/views/order_views.py [hunk 1]
```diff
@@ -15,21 +15,34 @@
...
 
+class TokenLoginRequiredMixin(LoginRequiredMixin):
+    """A login required mixin that allows token authentication."""
+
+    def dispatch(self, request, *args, **kwargs):
+        """If token was provided, ignore authenticated status."""
+        http_auth = request.META.get("HTTP_AUTHORIZATION")
+
+        if http_auth and "JWT" in http_auth:
+            pass
```
Comment (morento101) [inline beauty/api/views/order_views.py:37]: Useless if statement, or not?

File: beauty/api/tests/test_serializers_customuser.py [hunk 2]
```diff
@@ -276,3 +245,36 @@ def test_password_null(self):
...
+    """Password resetting tests.
+
+    Tests for checking password and password confirmation when user
+    resets password.
+    """
+
+    valid_data = {"password": "0967478911m", "confirm_password": "0967478911m"}
+    invalid_data = {"password": "0967478911m", "confirm_password": "096747891"}
+    null_data_one = {"password": "", "confirm_password": "096747891"}
+    null_data_two = {"password": "0967478911m", "confirm_password": ""}
```
Comment (morento101) [inline beauty/api/tests/test_serializers_customuser.py:260]: If you use data only once in one test, it is better to keep this data in this test method, not in setUp

setUp will generate this data too many times and tests will take a too long time

**Review by datsiuk-ira — APPROVED** [review]

File: beauty/api/views/order_views.py (hunk 1 above)
Comment (Misha86) [reply beauty/api/views/order_views.py:37]: It checks if the token exists

**Review by morento101 — APPROVED** [review]

File: beauty/api/views/order_views.py (hunk 1 above)
Comment (morento101) [reply beauty/api/views/order_views.py:37]: ok, I see, but you can then rewrite these ifs:
```py
if http_auth and "JWT" in http_auth:
            pass

elif not request.user.is_authenticated:
            return self.handle_no_permission()
```
```py
if not (http_auth and "JWT" in http_auth) and not request.user.is_authenticated:
            return self.handle_no_permission()
```
But this looks quite complicated

**Pull request merged**